Rohit/HTML basics remake/form.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>form</title>
+</head>
+<body>
+    <h1>Register</h1>
+ <form> 
+    <label for="First name">First name:</label> 
+    <input name="first name" type="text" placeholder="First name" required />
+    <label for="last name">Last name:</label> 
+    <input name="last name" type="text" placeholder="Last name" required /><br>
+    <br>
+    <label for="gender">gender:</label> 
+    <label for="male">Male</label>
+    <input  name="gender" type="radio" value="male" checked />
+    <label for="female">Female</label> 
+    <input  name="gender" type="radio" value="female" />
+    <label for="others">Others</label>
+    <input  name="gender" type="radio" value="others" /> <br>
+    <br>
+    <label for="email">Email:</label>
+    <input  name="Email" type="email" id="email" size="30" placeholder="Email"  required/>
+    <label for="password">Password:</label>
+    <input  name="password" type="password" minlength="5" maxlength="15" placeholder="Password" required/>
+    <label for="birthday">Birthday:</label>
+    <input  name="birthday" type="date"/> 
```

Comment:
Missing autocomplete. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)